packages/dd-trace/src/priority_sampler.js

```diff
@@ -137,6 +137,8 @@ class PrioritySampler {
   }
 
   _normalizeRules (rules, sampleRate) {
+    rules = [].concat(rules || [])
```

Comment:
What happens if rules is `null` or an invalid value?

Reply:
null sets it as no rules: https://github.com/DataDog/dd-trace-js/pull/2202/files#diff-e6d6fecb8b6063e1589f9833d990a9d1d4caa2cecdbeaf8e981748eb1c876ae5R248

As for invalid values, it can only come from our config object so any invalid inputs would be a bug on our part.